### PR TITLE
Fix for issue 92

### PR DESCRIPTION
**Pull request by leGoof**
Am I doing this right? It feels like I'm not.

Anyways, pull fix for #92 to the main branch.

### CODE REVIEW

**Comment by eaburns** [comment]
Close. `where` evaluates and returns an address. Since all commands set dot, it also sets dot to the address. But it seems like a better fit would be too call `mark('.')`. This makes it explicit that you are setting dot.

Also, your change for k isn't quite right. It would be better for `parseRune` to return an error instead of expecting `mark` to check the range and error out a few calls later. However, I'm wondering if we even want to get rid of this behavior of k. It's not hurting anything, and there is already a precedence in sam for having default values when things are left out. Dot seems like a good default here let's leave it.

**Comment by leGoof** [comment]
No longer calling where instead just setting ed.marks['.']. 
parseRune now returns an error; if nothing is provided we will return '.', if a none mark is entered (i.e. not a-z,A-Z) we will return an error.

**Comment by eaburns** [comment]
Looks good. Can you please update the comment on Edit to mention that an address without a command just sets dot? The remark should go first in the list of commands.

Also, this is a small change that should just be one commit. Can you please squash it all and add "Fixes #92" to the end of the commit message. When guthub sees that it will close the issue.

**Comment by eaburns** [comment]
And, sorry to nitpick, can you make the commit message say what the fix is? The first line should be a single sentence description of the change. Maybe say, "An Edit command with only an address sets dot." Then subsequent lines can add more detail (probably not needed here, just add the Fixes #92 comment).

**Comment by leGoof** [comment]
Third time's the charm?

**Comment by eaburns** [comment]
One more nitpick. All of the other command comments on Edit are in active voice (sets dot, not dot is set) and are indented with a tab.

Sorry if this seems so pedantic, but you'll get the hang of my complaints and it's very nice to have everything consistent.

**Comment by eaburns** [comment]
Sorry another. Can you make the second line of the commit message the first instead. I.e., don't say "A fix for..." When reading the history summary (the first line is the summary) we aren't going to remember what #92 is. Better to say what the change does on the first line. Then after the entire message, on the last line, separated from the other lines with one blank like, say "Fixes #92." Github understands that syntax and will close the issue along with the commit.

**Comment by eaburns** [comment]
Here's an example: https://github.com/eaburns/T/commit/1d8ab40525b5a02ee52973042c3105a32923c9eb

**Comment by eaburns** [comment]
I guess I've also been leaving a blank line after the first to make it more clear that is a stand-alone summary. But I'm not sure that's too important.

**Comment by eaburns** [comment]
Oops, and i swear this is my last. The commit line about mark with no names mark seeing dot isn't needed. That was the behavior before this commit too. This cleans it up.

**Comment by eaburns** [comment]
r has a rant about descriptions here: https://groups.google.com/d/topic/golang-dev/6M4dmZWpFaI/discussion

You're isn't bad like those he was ranting about, but it's a fun rant and, more importantly, he gives a good example description.

**Comment by leGoof** [comment]
Fourth time's the charm?

File: edit/edit.go [hunk 1]
```diff
@@ -520,6 +520,8 @@ func match(ed *Editor, at addr, re *re1.Regexp) ([][2]int64, error) {
 // Items in {} are optional.
 //
 // Commands are:
+//	addr
+//		addr sets the address of Dot.
```
Comment (eaburns) [inline edit/edit.go]: I would've just said "Sets the address of Dot."
The "addr" comes from the word addr on the previous line.

**Comment by eaburns** [comment]
Just that one comment, then you can click merge.

**Comment by leGoof** [comment]
ok all set. Though I'm pretty sure I don't have write access to actually merge...

**Comment by eaburns** [comment]
Oh. Oops. I'll get it tomorrow morning and I'll give you access too.

**Pull request merged**